ksql-functional-tests/src/test/resources/query-validation-tests/joins.json

```diff
@@ -1816,6 +1816,210 @@
         {"topic": "OUTPUT", "key": 0, "value": 1, "timestamp": 10},
         {"topic": "OUTPUT", "key": 0, "value": null, "timestamp": 20}
       ]
+    },
+    {
+      "name": "stream stream left join - invalid join field - contains literal",
```

Comment:
It's good to get in the habit of trimming down your test to only include the bits needed. This makes the test much easier to scan and understand. e.g. `key` field is not needed in with clause, only one field is needed in the schemas of the sources, and the join can just select everything. Putting that together you get:

```json
       "statements": [
        "CREATE STREAM TEST1 (ID bigint) WITH (kafka_topic='left_topic', value_format='JSON');",
        "CREATE STREAM TEST2 (ID bigint) WITH (kafka_topic='right_topic', value_format='JSON');",
        "CREATE STREAM OUTPUT as SELECT * FROM test1 t join Test2 tt WITHIN 11 seconds ON t.id = 0;"
```